changed incorrect sections (#6770)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/security-policy-settings/network-security-ldap-client-signing-requirements.md

### Best practices

> -   Set both **Network security: LDAP client signing requirements** and **Domain controller: LDAP server signing requirements** settings to **Require signing**. To avoid usage of unsigned treaffic, set both cleint and server sides to require signing. Not setting one of the sides will prevent client computers from communicating with the server. This can cause many features to fail, including user authentication, Group Policy, and logon scripts.

Reviewer: ```suggestion
-   Set both **Network security: LDAP client signing requirements** and **Domain controller: LDAP server signing requirements** settings to **Require signing**. To avoid usage of unsigned traffic, set both client and server sides to require signing. Not setting one of the sides will prevent client computers from communicating with the server. This can cause many features to fail, including user authentication, Group Policy, and logon scripts.
```

### Countermeasure

Configure the **Network security: LDAP client signing requirements** setting to **Require signing**.

### Potential impact

> If you configure the client to require LDAP signatures, it may fail to compunicate the LDAP servers that doesn't require the signing of requests. To avoid this issue, make sure that both **Network security: LDAP client signing requirements** and **Domain controller: LDAP server signing requirements** settings are set to **Require signing**.

Reviewer: ```suggestion
If you configure the client to require LDAP signatures, it may fail to communicate with the LDAP servers that do not require requests to be signed. To avoid this issue, make sure that both the **Network security: LDAP client signing requirements** and **Domain controller: LDAP server signing requirements** settings are set to **Require signing**.
```
